src/main/java/dev/jbang/cli/BaseCommand.java

```diff
@@ -30,7 +30,7 @@ public abstract class BaseCommand implements Callable<Integer> {
 	public static final int EXIT_INTERNAL_ERROR = 4;
 	public static final int EXIT_EXECUTE = 255;
 
-	private static final Logger logger = Logger.getLogger("org.jboss.shrinkwrap.resolver");
```

Comment:
Why this change? That logger was specifically there to shut up all the messages from the resolver. Or is that now not necessary anymore because of other changes you made? (The result should be that if you now wun with `--verbose` you could be getting a flood of resolver messages drowning out everything else)